src/google-maps/map-ground-overlay/map-ground-overlay.ts

```diff
@@ -46,7 +49,13 @@ export class MapGroundOverlay implements OnInit, OnDestroy {
   }
 
   /** Bounds for the overlay. */
-  @Input() bounds: google.maps.LatLngBounds|google.maps.LatLngBoundsLiteral;
+  @Input()
+  get bounds(): google.maps.LatLngBounds|google.maps.LatLngBoundsLiteral {
```

Comment:
I think it will be confusing to have both a "bounds" getter as well as the original "getBounds" method below. The pattern I have following has been to call get<property> on the underlying object instead of what is stored on the BehaviorSubject.

Reply:
We need the getter for backwards compatibility since `bounds` used to be a regular property before.

Reply:
Ah, that makes sense.